package.json

```diff
@@ -126,6 +126,7 @@
     "extract-text-webpack-plugin": "0.8.2",
     "file-loader": "0.8.4",
     "font-awesome": "4.4.0",
+    "flot-charts": "^0.8.3",
```

Comment:
Mind removing the version range specifiers here and elsewhere? If they were installed within the project they would have used the [.npmrc](https://github.com/elastic/kibana/blob/master/.npmrc) file and been saved to the exact version.